commands/version/__tests__/version-command.test.js

```diff
@@ -563,4 +563,17 @@ describe("VersionCommand", () => {
       });
     });
   });
+
+  describe("--include-merged-tags", () => {
+    it("accepts --include-merged-tags", async () => {
+      const testDir = await initFixture("normal");
+      await lernaVersion(testDir)("--include-merged-tags", "--yes", "patch");
+
+      expect(PromptUtilities.select).not.toBeCalled();
+      expect(PromptUtilities.confirm).not.toBeCalled();
+
+      const message = await getCommitMessage(testDir);
+      expect(message).toBe("v1.0.1");
```

Comment:
This isn't actually testing `--include-merged-tags`, though. I'd rather the tests you added to `changed-command.test.js` be moved to a separate test file under `commands/version/__tests__`, like `version-include-merged-tags.test.js`.